src/containers/PrivacyPolicy/PrivacyPolicy.scss

```diff
@@ -0,0 +1,54 @@
+.PrivacyPolicy {
+  color: var(--color-sail-gray-600);
+
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 60px;
+  max-width: 720px;
+  min-height: 700px;
+  padding-bottom: 120px;
+
+  h1 {
+    color: var(--color-sail-gray-900);
+  }
+
+  h2 {
+    color: var(--color-sail-gray-900);
+    font-size: 20px;
+    font-weight: 500;
+    margin-top: 40px;
+  }
+
+  h2:first-of-type {
+    margin-top: 80px;
+  }
```

Comment:
Hey, can you update these selectors? We shouldn't be targeting HTML elements directly. Use class names instead.

Reply:
Updated for both